pkg/httpclient/http_client.go

```diff
@@ -17,12 +18,17 @@ type httpClient struct {
 	headers map[string][]string
 }
 
-func NewHTTPClient(timeout time.Duration, overrideHeaders map[string][]string) Caller {
+func NewHTTPClient(timeout time.Duration, overrideHeaders map[string][]string, skipTLS bool) Caller {
 	transport := &http.Transport{
 		DialContext:     (&net.Dialer{Timeout: timeout}).DialContext,
 		IdleConnTimeout: timeout,
 	}
 
+	//TODO: add loki tls config https://issues.redhat.com/browse/NETOBSERV-309
+	if skipTLS {
+		transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
May be we could add a warning log here. Stating that TLS checks have been disabled.

Reply:
sure let's do this